AmericanMedicalAssociation/AMA-style-guide
# AMA-83: updates the Hero block twig files and adds BEM classes.

https://palantir.atlassian.net/browse/AMA-83

* Renames the Hero block, updates twig files, adds BEM-friendly classes
* Removes the read more link atom

Noted in files:

1. Widths are for placeholder only and will need to be updated with layout.
2. Fonts use placeholder classes that will be replace wiht semantic classes or mixin.
3. Responsive typogray will happen with implementation of breakpoints.

### Steps to test:

1. Navigate to Molecules > Blocks > Hero Block
2. Observe that the Hero Block is present, is styled properly, and behaves correctly on hover
